### ISSUE TITLE
Overkill it but i think I miss something

### ISSUE THREAD

**Issue by cordoval** [cordoval/FormModelBundle#2]
@everzet I have overkilled this example for the calculator check under Tests folder and Features folder and Class Calculator

now i understood what you were saying. So the problem is now to have more practice, do you recommend a way to make this example more complex a bit and make this beneficial to the average user? Say wonder if it is ok to implement the model form that i was talking about anyway

please let me know your thoughts, I will blog about it soon.

so far spec bdd and scen bdd worked very similar up to a point, which should be first?

And i believe still that spec bdd even though in phpunit mainly can be aided by behat scenarios.... or perhaps! let-s work on a bundle for allowing PhpSpecBundle would you please?

**Comment by mdpatrick** [comment]
Did you end up making a blog post on this, Luis???

**Comment by cordoval** [comment]
yes I did, I am actually using it for a live example but will come after because i will have to simplify it, but practice is practice so get practice

**Comment by mdpatrick** [comment]
Can you link me to the post :) I'd like to read more about what's going on here
